crowbar_framework/app/controllers/api/storages_controller.rb

```diff
@@ -15,6 +15,8 @@
 #
 
 class Api::StoragesController < ApiController
+  before_action :set_storage
```

Comment:
(completely out-of-topic, but I think we should have named everything "Ceph", not "Storage", as the repo is called crowbar-ceph)